### PR TITLE
Kds 1855 remove unused dependencies

### PR DESCRIPTION
**Pull request by gyfchong**
## Notes

- CSS layers help with the layout shifts we're seeing in Chromatic
- Apply CSS vars for tokens to `:root`
- Include reset + normalize in KAIO styles (supporting Tailwind borders) reducing setup steps for consumers

### CODE REVIEW

File: storybook/manager.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import React from "react"
 import { addons } from "@storybook/addons"
 import { API_SidebarOptions as SidebarOptions } from "@storybook/types"
-import { defaultTheme } from "@kaizen/design-tokens"
+import { tokens } from "../packages/design-tokens/src/js"
```
Comment (gyfchong) [inline storybook/manager.tsx:6]: Not sure why `@kaizen/design-tokens/js` doesn't work...

Comment (HeartSquared) [reply storybook/manager.tsx:6]: Would `~design-tokens/js` work? Seems we had made an alias

Comment (gyfchong) [reply storybook/manager.tsx:6]: Also doesn't work, I suspect we have a workspaces issue...

Comment (gyfchong) [reply storybook/manager.tsx]: Works as a package again

File: packages/design-tokens/src/index.ts [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
+
+/**
+ * @deprecated
+ * - Not needed if you are using `KaizenProvider` from `@kaizen/components` or `defaultPreset` from next-services.
+ * - If you require direct usage of these tokens in js you can `import { tokens } from "@kaizen/design-tokens/js"`
+ */
 export * from "./themes"
+
+/**
+ * @deprecated
```
Comment (HeartSquared) [inline packages/design-tokens/src/index.ts:11]: Oh? Does this apply the comment to everything exported?

File: packages/hosted-assets/README.md [hunk 3]
```diff
@@ -27,6 +27,7 @@ asset-url("some/blob.png") // -> "https://<origin>/some/blob.png"
 
 ```js
 // TS/JS
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (HeartSquared) [inline packages/hosted-assets/README.md:30]: You need linting for docs?? :s

File: yarn.lock [hunk 4]
```diff
@@ -2706,29 +2706,7 @@
...
-"@kaizen/brand@^1.5.8":
-  version "1.5.8"
-  resolved "https://registry.yarnpkg.com/@kaizen/brand/-/brand-1.5.8.tgz#934d90e710c3f82b173e06c58111215c3bca40d6"
-  integrity sha512-XGbgwhyBUNjF7/GONBHKpEmVar6TfiTYu7sU/tLzcjQroasa9bf0NBf4Lp0MhUHSXXHShRX1tdRW038zfDtCFQ==
-  dependencies:
-    "@kaizen/component-base" "^1.1.7"
-    "@kaizen/hosted-assets" "^2.0.3"
-
-"@kaizen/button@^3.0.4", "@kaizen/button@^3.0.5":
+"@kaizen/button@^3.0.4":
```
Comment (HeartSquared) [inline yarn.lock:2709]: If all components are migrated before this is merged, we can check the yarn.lock no longer has any legacy deps.

**Review by HeartSquared — COMMENTED** [review]
Reminder: Revise Chromatic CI to trigger when any `.css` file is updated (currently only looks for `.scss`)

File: packages/design-tokens/src/index.ts (hunk 2 above)
Comment (gyfchong) [reply packages/design-tokens/src/index.ts:11]: I thought it did, but apparently I never tested this and it does not work haha

**Review by gyfchong — COMMENTED** [review]
Throwaway

**Review by gyfchong — COMMENTED** [review]
Throwaway

File: packages/design-tokens/src/index.ts (hunk 2 above)
Comment (gyfchong) [reply packages/design-tokens/src/index.ts:11]: Need some way to deprecate external usage...

File: .github/workflows/test-stories.yaml [hunk 5]
```diff
@@ -116,8 +116,7 @@ jobs:
           externals: |
             [
               "**/!(*.module).scss",
-              "packages/button/src/Button/*.scss",
-              "packages/component-library/components/Spacing/Base.module.scss"
+              "packages/components/**/*.css",
```
Comment (HeartSquared) [inline .github/workflows/test-stories.yaml:119]: Wasn't sure if we wanted it to be a blanket one, so only applied it to KAIO. Can revise.

File: packages/components/src/Icon/ArrowDownIcon.tsx [hunk 6]
```diff
@@ -1,18 +1,17 @@
...
 export const ArrowDownIcon = (props: IconProps): JSX.Element => {
-  const uniqueId = uuidv4()
+  const uniqueId = useId()
   const svgContent = (
     <>
       <defs>
         <path
           id={uniqueId}
-          d="m16 8-1.057-1.058-4.193 4.185V2h-1.5v9.127L5.065 6.935 4 8l6 6z"
+          d="m16 8-1.057-1.057-4.193 4.185V2h-1.5v9.128L5.065 6.935 4 8l6 6z"
```
Comment (HeartSquared) [inline packages/components/src/Icon/ArrowDownIcon.tsx:14]: This is a strange change :s

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**